### DISCUSSION TITLE
customizing the sidebar.

### DISCUSSION THREAD

**Discussion by DeepKumbhare85** [vuejs/vitepress#4305] [Q&A]
I want to use Icon in my sidebar Navigation text. I have created a vue component (SidebarItem) for it. but using it inside text prop doesn't work. 
```ts
 { text: '<SidebarItem :title="installation">', link: '/guide/installation' },
```
Is there something I am missing or doing something wrong?

**Comment by brc-dd** [comment]
This isn't supported. Only normal html is supported.
